symbolic-debuginfo/src/wasm.rs

```diff
@@ -113,16 +110,16 @@ impl<'d> WasmObject<'d> {
     }
 
     /// Returns an iterator over symbols in the public symbol table.
-    pub fn symbols<'o>(&'o self) -> WasmSymbolIterator<'d, 'o> {
+    pub fn symbols(&self) -> WasmSymbolIterator<'data, '_> {
+        let iterator = Box::new(self.wasm_module.funcs.iter()) as Box<dyn Iterator<Item = _>>;
```

Comment:
is it possible to un-box this?

Reply:
I'm not sure. Walrus returns an `impl Iterator`.